react/OrderItems.tsx

```diff
@@ -34,6 +34,14 @@ const enum Totalizers {
   DISCOUNT = 'Discounts',
 }
 
+interface AddItemOptions {
+  allowedOutdatedData?: string[]
+}
+
+interface UpdateItemOptions {
+  allowedOutdatedData?: string[]
+}
```

Comment:
As we conversed, it's better (maybe not best?) to do it something like
```typescript
type AddItemOptions = Pick<MutationAddToCartArgs, 'allowedOutdatedData'>
```

Reply:
@wdsrocha is it still valid? @silviasfon, will you make this change?